### DISCUSSION TITLE
Shoutout command

### DISCUSSION THREAD

**Discussion by Chico3tm** [Sehelitar/Kick.bot#12] [Q&A] [answered]
Has anyone got shoutout commands to work?? I got everything set up I think, but when I test the command it only pulls the variables for the person doing the shoutout. I've posted a picture of the command and everything I have setup. And I posted the return of the command. BotRix responds correctly but my command in streamer.bot not it shoutout the person doing the shoutout

![bot](https://github.com/Sehelitar/Kick.bot/assets/147455603/c1d6adc9-3e71-49b7-8a04-285ae36794e9)

![streamerbot](https://github.com/Sehelitar/Kick.bot/assets/147455603/d9fb6218-8bcf-4a56-bb49-517ebae3af81)

**Comment by Sehelitar** [comment]
Hey 👋
You have to add a "**Set argument**" sub-action in the first position to set the argument ``targetUser`` to the value of ``%input0%``.
You can then use ``%targetUser%`` and ``%targetUserName%`` in your message.

``%user%`` and ``%userName%`` will always refer to the person sending the command, not the one mentioned **in** the message.

**Reply by Chico3tm** [reply]
Thanks a lot, I'm still a little lost lol I'm new to this but I'll try to understand what you saying and play with it. At least you gave me the right values to use. I've been working on this for over 72hrs lol

**Reply by Sehelitar** [reply, marked as answer]
You should have something like this. I also added a check to see if you didn't forget the person's name (in case you do ``!so`` without anything after)

![image](https://github.com/Sehelitar/Kick.bot/assets/1716704/c8131897-d854-4865-99da-1c910e190af7)

**Reply by Chico3tm** [reply]
Thanks a lot I was not even close to what you have above.
